docs/tutorials/rerank/rerank_pipeline_with_Amazon_Rerank_model_on_Amazon_Bedrock.md

```diff
@@ -0,0 +1,660 @@
+
+This tutorial illustrates using the [Amazon Rerank 1.0 model in Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/rerank-supported.html) in a reranking pipeline. 
+
+Note: Replace the placeholders that start with `your_` with your own values.
+
+# Steps
+
+## 0. Test the model on Amazon Bedrock
+You can perform a reranking test using the following code.
+
+```python
+import json
+import boto3
+bedrock_region = "your_bedrock_model_region_like_us-west-2"
+bedrock_runtime_client = boto3.client("bedrock-runtime", region_name=bedrock_region)
+
+modelId = "amazon.rerank-v1:0"
+contentType = "application/json"
+accept = "*/*"
+
+body = json.dumps({
+    "query": "What is the capital city of America?",
+    "documents": [
+        "Carson City is the capital city of the American state of Nevada.",
+        "The Commonwealth of the Northern Mariana Islands is a group of islands in the Pacific Ocean. Its capital is Saipan.",
+        "Washington, D.C. (also known as simply Washington or D.C., and officially as the District of Columbia) is the capital of the United States. It is a federal district.",
+        "Capital punishment (the death penalty) has existed in the United States since beforethe United States was a country. As of 2017, capital punishment is legal in 30 of the 50 states."
+    ]
+})
+
+response = bedrock_runtime_client.invoke_model(
+    modelId=modelId,
+    contentType=contentType,
+    accept=accept, 
+    body=body
+)
+results = json.loads(response.get('body').read())["results"]
+print(json.dumps(results, indent=2))
+```
+
+The reranking result is ordering by the highest score first:
+```
+[
+  {
+    "index": 2,
+    "relevance_score": 0.7711548724998493
+  },
+  {
+    "index": 0,
+    "relevance_score": 0.0025114635138098534
+  },
+  {
+    "index": 1,
+    "relevance_score": 2.4876490010363496e-05
+  },
+  {
+    "index": 3,
+    "relevance_score": 6.339210403977635e-06
+  }
+]
+```
+
+
+You can sort the result by index number.
+
+```python
+print(json.dumps(sorted(results, key=lambda x: x['index']),indent=2))
+```
+
+The results are as follows:
+```
+[
+  {
+    "index": 0,
+    "relevance_score": 0.0025114635138098534
+  },
+  {
+    "index": 1,
+    "relevance_score": 2.4876490010363496e-05
+  },
+  {
+    "index": 2,
+    "relevance_score": 0.7711548724998493
+  },
+  {
+    "index": 3,
+    "relevance_score": 6.339210403977635e-06
+  }
+]
+```
+
+## 1. Create a connector and register the model
+
+To create a connector for the model, send the following request. If you are using self-managed OpenSearch, supply your AWS credentials:
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "Amazon Bedrock cross-encoder model",
```

Comment:
```suggestion
  "name": "Amazon Bedrock rerank model",
```

Reply:
cross-encoder is more technical , suggest change to `rerank` to avoid confusion to user who don't have knowledge about cross-encoder
```
"model_name": "amazon.rerank-v1:0"
```